chain/jsonrpc-primitives/src/utils.rs

```diff
@@ -3,7 +3,7 @@ use serde_json::Value;
 
 use near_primitives::borsh::BorshDeserialize;
 
-pub(crate) fn parse_params<T: DeserializeOwned>(
+pub fn parse_params<T: DeserializeOwned>(
```

Comment:
Please, revert since a copy of this implementation lives in `near-jsonrpc` behind `adversarial` feature (we consider how to refactor that, but we don't want to expose hacky tools across crates boundary, so this impl should remain to be `pub(crate)`

Reply:
removed